api/sheets/index.md

```diff
@@ -8,50 +8,75 @@ has_children: true
 
-**"Add-ons > Data Commons > Fill place dcids"**
 
-![](/assets/images/sheets/sheets_menu_bar.png)
+## Finding a Place’s DCID
 
-A sidebar will appear on the right of the sheet, with a search bar where you can start typing the name of the place you want. From the drop down menu, select the place you want, and its [DCID](/glossary.html) will appear in the cell that you selected.
+Data Commons’ Sheets add-on ships with the ability to look up a location’s [DCID](https://docs.datacommons.org/glossary.html), within the application by using the **Fill place dcids** feature. To use this feature to find a place’s DCID:
 
-![](/assets/images/sheets/sheets_search_box.png)
+1. Select the destination cell where you want to add a place’s [DCID](https://docs.datacommons.org/glossary.html).
+2. Click in the search box for the **Fill place dcids for selected cells** section.
+3. Start typing the name of the place you are searching for. From the drop-down menu, select the place you want, and its [DCID](https://docs.datacommons.org/glossary.html) appears in the cell that you selected. For example, the following image shows the place names that match “Hawaii”.
+   ![](/assets/images/sheets/sheets_finding_dcid.png)
 
 Note that this feature only supports places in the United States.
 
-## Sorting your results
+## Introduction to the Sheets API Custom Functions
+
+The Data Commons Sheets API includes the five formulas listed in the following table. Click the links in the table for detailed information on each formula.
+
+| **Formula**                                                                                  | **Description**                           |
+|----------------------------------------------------------------------------------------------|-------------------------------------------|
+| [=DCGETNAME(_dcids_)](https://docs.datacommons.org/api/sheets/get_name.html)                 | Returns the names associated with a DCID. |
+| [=DCPLACESIN(_dcids_)](https://docs.datacommons.org/api/sheets/places_in.html)               | Retrieves child nodes                     |
+| [=DCGET(_dcids_)](https://docs.datacommons.org/api/sheets/get_variable.html)                 | Returns statistical variables.            |
+| [=DCPROPERTY(_dcids_)](https://docs.datacommons.org/api/sheets/get_property.html)            | Retrieves node property values.           |
+| [=DCCOHORTMEMBERS(_dcids_)](https://docs.datacommons.org/api/sheets/get_cohort_members.html) | Gets the cohort members of a node.        |
+
+See the links in the preceding table to subsequent pages in this section for detailed descriptions of each API.
 
-<p>To sort columns of data from the API, we suggest using the "filter views" Google Sheet feature.
+## Sorting your Results
 
-<p>Please check the following video for a complete example:</p>
+To sort columns of data from the API, consider using Google Sheets “filter views” feature, which allows you to alphabetically and numerically sort and filter data
```

Comment:
```suggestion
To sort columns of data from the API, use the Google Sheets “filter views” feature, which allows you to alphabetically and numerically sort and filter data
```

Reply:
This change hasn't been made

Reply:
Drop the word "consider" since this is the only way to sort.

Reply:
I fixed it.